pybaseball/playerid_lookup.py

```diff
@@ -1,19 +1,26 @@
-import pandas as pd
-import requests
 import io
 import os
 
+import pandas as pd
+import requests
+
+from .datahelpers import caching
+
 # dropped key_uuid. looks like a has we wouldn't need for anything.
 # TODO: allow for typos. String similarity?
 
 url = "https://raw.githubusercontent.com/chadwickbureau/register/master/data/people.csv"
-register_file = 'chadwick-register.csv'
 
+def get_register_file():
+    return os.path.join(caching.cache_config.cache_directory, 'chadwick-register.csv')
+
+
+@caching.dataframe_cache()
 def chadwick_register(save: bool = False) -> pd.DataFrame:
     ''' Get the Chadwick register Database '''
 
-    if os.path.exists(register_file):
-        table = pd.read_csv(register_file)
+    if os.path.exists(get_register_file()):
+        table = pd.read_csv(get_register_file())
         return table
```

Comment:
I think this makes sense separately. Perhaps, even pull from the GitHub?

Reply:
Possibly as a future improvement. For now adding a decent cache with two lines of code seems like a win.

Reply:
Makes sense.